docs/notebooks/Topic_dendrogram.ipynb

```diff
@@ -0,0 +1,378 @@
+    "        topic_vals.pop(trace[0], None)\n",
+    "        topic_vals.pop(trace[2], None)  \n",
+    "        \n",
+    "    return text_annotations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true,
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "# get text annotations\n",
+    "annotation = text_annotation(topic_dist, topic_terms, n_ann_terms)\n",
+    "\n",
+    "# Plot dendrogram\n",
+    "dendro = create_dendrogram(topic_dist, distfun=js_dist, labels=range(1, 36), annotation=annotation)\n",
+    "dendro['layout'].update({'width':1000, 'height':600})\n",
+    "py.iplot(dendro)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The x-axis or the leaves of hierarchy represent the topics of our LDA model, y-axis is a measure of closeness of either individual topics or their cluster. Essentially, the y-axis level at which the branches merge (relative to the \"root\" of the tree) is related to their similarity. For ex., topic 15 and 27 are more similar to each other than to topic 25. In addition, topic 8 and 30 are more similar to 28 than topic 15 and 27 are to topic 25 as the height on which they merge is lower than the merge height of 15/27 to 25.\n",
+    "\n",
+    "Text annotations visible on hovering over the cluster nodes show the intersecting/different terms of it's two child nodes. Cluster node on first hierarchy level uses the topics on leaves directly to calculate intersecting/different terms, and the upper nodes assume the intersection(+++) as the topic terms of it's child node."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Dendrogram with a Heatmap\n",
+    "\n",
+    "Now lets append the distance matrix of the topics below the dendrogram in form of heatmap after which we would be able to see the exact distances between all pair of topics."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# get text annotations\n",
+    "annotation = text_annotation(topic_dist, topic_terms, n_ann_terms)\n",
+    "\n",
+    "# Initialize figure by creating upper dendrogram\n",
+    "figure = create_dendrogram(topic_dist, distfun=js_dist, labels = range(1, 36), annotation=annotation)\n",
+    "for i in range(len(figure['data'])):\n",
+    "    figure['data'][i]['yaxis'] = 'y2'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# get distance matrix and it's topic annotations\n",
+    "mdiff, annotation = lda_fake.diff(lda_fake, distance=\"jensen_shannon\", normed=False)\n",
+    "\n",
+    "# get reordered topic list\n",
+    "dendro_leaves = figure['layout']['xaxis']['ticktext']\n",
+    "dendro_leaves = list(map(int, dendro_leaves-1))\n",
```

Comment:
It's a list of int values where every value needs to be subtracted by 1. Simplified this using list comprehension now